pkg/app/image_describe.go

```diff
@@ -50,6 +48,6 @@ func getImageDependency(clicontext *cli.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to list dependencies")
 	}
-	renderDependencies(dep, os.Stdout)
```

Comment:
It is for test purposes, should we keep it?

Or it is hard to test the feature in the future.

Reply:
Do you mean writing to `stdout` is for test purposes?

Even now, testing this feature is problematic unless we support output JSON.

Reply:
I guess gaoce's meaning is that we shoud pass the fd currently?

Reply:
We may write UT for `renderDependencies`

Reply:
Fixed. PTAL.

Reply:
LGTM